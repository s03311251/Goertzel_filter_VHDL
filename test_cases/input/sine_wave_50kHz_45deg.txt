36A0
3C82
3F9A
3F9A
3C82
36A0
2E86
2501
1AFE
1179
095F
037D
0065
0065
037D
095F
1179
1AFE
2501
2E86
36A0
3C82
3F9A
3F9A
3C82
36A0
2E86
2501
1AFE
1179
095F
037D
0065
0065
037D
095F
1179
1AFE
2501
2E86
36A0
3C82
3F9A
3F9A
3C82
36A0
2E86
2501
1AFE
1179
095F
037D
0065
0065
037D
095F
1179
1AFE
2501
2E86
36A0
3C82
3F9A
3F9A
3C82
36A0
2E86
2501
1AFE
1179
095F
037D
0065
0065
037D
095F
1179
1AFE
2501
2E86
36A0
3C82
3F9A
3F9A
3C82
36A0
2E86
2501
1AFE
1179
095F
037D
0065
0065
037D
095F
1179
1AFE
2501
2E86
