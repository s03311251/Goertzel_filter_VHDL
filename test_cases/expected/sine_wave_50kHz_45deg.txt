FFF1B620
FFEDFC40
